.travis.yml

```diff
@@ -13,5 +13,39 @@ addons:
       - libxerces-c-dev
       - gcc-4.8
       - g++-4.8
+      - gcc-5
+      - g++-5
+
+before_install:
+  - git clone https://github.com/appleseedhq/travis-linux-deps.git
+
+install:
+  - if [ -n "$GCC_VERSION" ]; then export CC="${CC}-${GCC_VERSION}"; fi
+  - if [ -n "$GCC_VERSION" ]; then export CXX="${CXX}-${GCC_VERSION}"; fi
+
+compiler:
+  - gcc
+
+env:
+    - GCC_VERSION=4.8 USE_CPP11=OFF
+
+matrix:
+    include:
+        - os: linux
+          compiler: gcc
+          env: GCC_VERSION= USE_CPP11=OFF
```

Comment:
Is `GCC_VERSION` set to empty on purpose? Is it to clear the env var? What version are we using then?